apps/settings/js/media_storage.js

```diff
@@ -538,7 +538,7 @@ require([
       this.defaultMediaLocation.addEventListener('click', this);
       this.makeDefaultLocationMenu();
 
-      window.addEventListener('localized', this);
+      document.addEventListener('DOMRetranslated', this);
```

Comment:
any document for that change? Though I can saw that in https://github.com/l20n/l20n.js/blob/master/src/runtime/gaia/index.js